OpenQA/Qemu/Proc.pm

```diff
@@ -335,6 +335,7 @@ sub init_blockdev_images {
         die "init_blockdev_images: '@$qicmd' failed after $tries tries: $@" if $@;
     }
 
+    die "reached init_blockdev_images after runcmd";
```

Comment:
This means the next line is dead-code. Is this a leftover from debugging?

Reply:
yes, replaced by "bmwqemu::diag" with a better message string